workspace/resource_directory.go

```diff
@@ -0,0 +1,56 @@
+package workspace
+
+import (
+	"context"
+	"errors"
+
+	"github.com/databrickslabs/terraform-provider-databricks/common"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// ResourceDirectory manages directories
+func ResourceDirectory() *schema.Resource {
+	s := map[string]*schema.Schema{
+		"path": {
+			Type:     schema.TypeString,
+			Required: true,
+			ForceNew: true,
+		},
+		"delete_recursive": {
+			Type:     schema.TypeBool,
+			Default:  false,
+			Optional: true,
+		},
+	}
+
+	return common.Resource{
+		Schema:        s,
+		SchemaVersion: 1,
```

Comment:
Schema version should be zero or omitted

Reply:
Great, will do that later today.